src/main/java/Taylor.java

```diff
@@ -0,0 +1,113 @@
+import java.util.*;
+
+public class Taylor {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        final String line = "____________________________________________________________";
+        System.out.println(line);
+        System.out.println("Hello! I'm Taylor");
+        System.out.println("What can I do for you?");
+        System.out.println(line);
+
+        String input = sc.nextLine();
+        List<Task> tasks = new ArrayList<>();
+        while(!input.equals("bye")) {
```

Comment:
For all the printing of the statements, maybe can put it in a method each. For instance, printTodoAdded() and the statement that the todo task have been added will be printed